redaxo/src/core/default.config.yml

```diff
@@ -80,3 +80,4 @@ accesskeys:
     delete: d
     add: a
     add_2: y
+system_editor: null
```

Comment:
Bin mir noch nicht so sicher, ob ich den Key gut finde.
Unter `system_editor` würde ich glaube ich eher verstehen, welcher Editor im System (REDAXO) verwendet werden soll. Also codemirror und so.
Oder finden die anderen das gut verständlich?
Andere Ideen: `editor`, `editor_url`
Bei Symfony heißt es `ide`: https://symfony.com/doc/current/reference/configuration/framework.html#ide
Aber da finde ich `editor` glaube ich besser

Reply:
Hmm, `editor_url` würde bei uns doch nicht passen, man trägt ja gar nicht die URL ein, sondern ein Kürzel für den Editor, und es wird intern in die richtige URL umgewandelt.

Reply:
ich habe system_ prefix genommen da die anderen settings die man hier unter der "system" page einstellt auch diesen prefix haben.

Reply:
hmm, welche meinst du da?
debug, lang, server, error_email .. die haben doch alle keinen Präfix?

Reply:
https://github.com/redaxo/redaxo/pull/1832/files#diff-707fa72572dccaeecc71e1e76ab94214R75